components/director/internal/open_resource_discovery/ord_document.go

```diff
@@ -138,19 +138,33 @@ func (docs Documents) Validate(calculatedBaseURL string, apisFromDB map[string]*
 		}
 	}
 
+	invalidApisIndices := make([]int, 0)
+	invalidEventIndices := make([]int, 0)
```

Comment:
change variable name to `invalidEventsIndices`. Do the same to the others bellow.